src/transition/kml_iss.cpp

```diff
@@ -0,0 +1,176 @@
+
+#include <kml_iss.h>
+
+#include <solver.h>
+
+
+static int sparse_err_t_2_int( const sparse_err_t err ) {
+
+	int result = 0;
+	switch ( err ) {
+		case NO_ERROR:
+		{
+			result = KMLSS_NO_ERROR;
+			break;
+		}
+		case NULL_ARGUMENT:
+		{
+			result = KMLSS_NULL_ARGUMENT;
+			break;
+		}
+		case ILLEGAL_ARGUMENT:
+		{
+			result = KMLSS_BAD_N;
+			break;
+		}
+		case OUT_OF_MEMORY:
+		{
+			result = KMLSS_NO_MEMORY;
+			break;
+		}
+		case UNKNOWN:
+		{
+			result = KMLSS_INTERNAL_ERROR;
+			break;
+		}
+	}
+	return result;
+}
+
+#define KML_CG_PREFIXED( name ) KmlIssCg ## name
+
+
+int KML_CG_PREFIXED(InitSI)( KmlSolverTask **handle, int n, const float *a, const int *ja, const int *ia ) {
```

Comment:
Please check for line overflow (80 chars max, starting from first non-space char)

Reply:
fixed